### PR TITLE
Add utm_source to enable some basic metrics collection in Pontoon

### PR DESCRIPTION
**Pull request by MikkCZ**

### CODE REVIEW

File: src/js/browser-action/browser-action.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function withRemotePontoon(remotePontoon) {
     // See all notifications
     document.querySelector('.notification-list .see-all').addEventListener('click', (e) => {
         e.preventDefault();
-        browser.tabs.create({url: remotePontoon.getNotificationsUrl()});
+        browser.tabs.create({url: `${remotePontoon.getNotificationsUrl()}?utm_source=pontoon-tools`});
```
Comment (mathjazz) [inline src/js/browser-action/browser-action.js]: Why not add the param to `getNotificationsUrl()` like for other Pontoon URLs?

Why not also add the param to `getSettingsUrl()`?

File: src/js/shared/RemotePontoon.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ class RemotePontoon {
      * @public
      */
     getTeamBugsUrl() {
-        return `${this._baseUrl}/${this._team}/bugs/`;
+        return `${this._baseUrl}/${this._team}/bugs/?utm_source=pontoon-tools`;
```
Comment (mathjazz) [inline src/js/shared/RemotePontoon.js]: That 404s sadly.

File: src/js/browser-action/browser-action.js (hunk 1 above)
Comment (MikkCZ) [reply src/js/browser-action/browser-action.js]: I have added the param only to links, that the user opens, so the automatic data updates are not included. `getNotificationsUrl()` is actually used for getting the notifications each 15 minutes and `getSettingsUrl()` for the team detection.

File: src/js/shared/RemotePontoon.js (hunk 2 above)
Comment (mathjazz) [reply src/js/shared/RemotePontoon.js]: Yes, that's a bug in how Pontoon handles dashboard tab URLs.

File: src/js/browser-action/browser-action.js (hunk 1 above)
Comment (MikkCZ) [reply src/js/browser-action/browser-action.js]: Would you prefer `utm_medium` (next level of the UTM hierarchy) to be used for distinguishing user and automation activity, or different `utm_source` value?

Comment (mathjazz) [reply src/js/browser-action/browser-action.js]: I don't know UTM hierarchy too well, but the 2nd option seems more appropriate.

File: src/js/shared/RemotePontoon.js (hunk 2 above)
Comment (MikkCZ) [reply src/js/shared/RemotePontoon.js]: Fixed now.

File: src/js/browser-action/browser-action.js (hunk 1 above)
Comment (MikkCZ) [reply src/js/browser-action/browser-action.js]: I have added `pontoon-tools-automation` value for those UI URL, that are used to pull data from. Note that GA won't show these at all and you have to track them on the server in logs. Should I add those for API and mark all notifications as read AJAX call?

Comment (mathjazz) [reply src/js/browser-action/browser-action.js]: Why won't GA show them? Does it ignore query strings?

Comment (MikkCZ) [reply src/js/browser-action/browser-action.js]: No, but when loading the data, the extension gets the HTML only. No JS is executed. You will see utm_source only for page opened in a tab by the user (e.g. by clicking a notification).

**Pull request merged**